sound/soc/sof/intel/cnl.c

```diff
@@ -314,22 +314,33 @@ int cnl_ipc_send_msg(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg)
 
 void cnl_ipc_dump(struct snd_sof_dev *sdev)
 {
-	u32 hipcctl;
```

Comment:
@ujfalusi Would it be better to define seaprate ipc_dump ops for IPC3 and IPC4 instead of using the same function and set them in ops_init() so that we dont need to keep the IPC version check in one place?

Reply:
@ranj063, hrm, yes, these are already callbacks.
Let me re-spin this series...